org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/DeviceRunner.java

```diff
@@ -114,7 +114,7 @@ public IPosition call() throws Exception {
 			try {
 				if (detector instanceof AbstractRunnableDevice) ((AbstractRunnableDevice)detector).setBusy(true);
 			    detector.run(position);
-			} catch (Exception ne) {
+			} catch (Throwable ne) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Catch Exception instead of Throwable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1181)